### DISCUSSION TITLE
How do you know if the user is logged on steam ?

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#234] [Q&A] [answered]
How do you know if the user is logged on steam ?

**Comment by lpeyr** [comment, marked as answer]
Gavilya can't know if you're logged on Steam. It can only know if Steam is running. When you launch a Steam game, Gavilya executes the following command: `steam://rungameid/{SteamAppIdHere}`. This command tells Steam to launch the game from its ID. However, if the Steam client isn't running, or if the user isn't logged in, nothing will happen, or an error will be shown to the user.

**Reply by sgamesdev** [reply]
This is what i did on my app , in the regestry Computer\HKEY_CURRENT_USER\SOFTWARE\Valve\Steam\ActiveProcess , ActiveProcess is 0 when the user isn't logged in , so if that value isn't 0 , it indicates that the user has logged in , that's it .

**Reply by lpeyr** [reply]
Thanks for the tip! I'll try to implement an error message if the user isn't logged in.

**Reply by sgamesdev** [reply]
RegistryKey key = Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Valve\Steam\ActiveProcess");
            string result = key.GetValue("ActiveUser").ToString();
            if(result == "0")
            {
                MessageBox.Show("Please login on steam or something ");
            }
